doc/pages/Manual.md

```diff
@@ -1202,6 +1202,10 @@ allowed.
   installs relative to `<prefix>/share/` (since <span class="opam">opam</span> 1.2.0)
 - <a id="installfield-etc">`etc:`</a>
   installs to `<prefix>/etc/<pkgname>/`
+- <a id="installfield-etc_root">`etc_root:`</a>
+  installs to `<prefix>/etc/` (since <span class="opam">opam</span> 2.0.6)
```

Comment:
Not sure what what version should be put here.

Reply:
better put `2.1.0`